.gitlab-ci.yml

```diff
@@ -187,7 +187,7 @@ onboarding_dotnet:
           ONBOARDING_FILTER_WEBLOG: [test-app-dotnet-container]
           SCENARIO: [ CONTAINER_AUTO_INJECTION_INSTALL_SCRIPT_PROFILING]
         - ONBOARDING_FILTER_ENV: [dev, prod]
-          ONBOARDING_FILTER_WEBLOG: [test-app-dotnet,test-app-dotnet-container]
+          ONBOARDING_FILTER_WEBLOG: [test-app-dotnet,test-app-dotnet-container,test-app-dotnet-multicontainer,test-app-dotnet-multialpine]
```

Comment:
If we use the variant "test-app-dotnet-multicontainer", we don't need to execute the variant "test-app-dotnet-container"
Could we remove the "test-app-dotnet-container" from the pipeline? only remove from the pipeline, don't remove the variant folders and files